server/config.ts

```diff
@@ -109,6 +109,15 @@ export default {
       },
       agent: new AgentConfig(Number(get('PRISON_REGISTER_API_TIMEOUT_RESPONSE', 10000))),
     },
+    // 30 second timeouts because bulk resetting VO balances can be slow
```

Comment:
Expecting resetting allocations to be pretty much a one-off per prison as they are migrated so not expecting long timeout to be a problem.